### DISCUSSION TITLE
How to contribute to the project

### DISCUSSION THREAD

**Discussion by yaten2302** [anuraghazra/github-readme-stats#3113] [Q&A] [answered]
#### Describe the Problem

Hi, I want to contribute to this project, can anyone please guide me that should I first learn GraphQL(as I'm a beginner, so I'm facing some problems while reading and understanding the code.)🙏🙏

**Comment by qwerty541** [comment, marked as answer]
There is not much GraphQL in the project code, most likely less than 5%, it is quite simple and intuitive. I personally never studied it separately, but somehow figured it out on the go. Do it the way you're comfortable with it. If you're having trouble reading the code, then it might be worth getting a deeper understanding of JavaScript and GraphQL syntax before trying to contribute.
